src/main/java/trello/entities/List.java

```diff
@@ -25,7 +25,7 @@
 
     private String id;
     private String title;
-    private ArrayList<Card> cards;
+    private java.util.List<Card> cards;
```

Comment:
Please go up this import to the section of import.

Reply:
This is not an import. It is like that so the IDE couldn't confuse between the java.util.List and the entity List that we are using in the context.